component/header.go

```diff
@@ -105,42 +102,37 @@ func (u *updatesHeading) Render() *vecty.HTML {
 	)
 }
 
+func (u *updatesHeading) updateAll(*vecty.Event) {
```

Comment:
Just personal preference - I prefer to extract handlers into their own methods, rather than declaring them inline.

Reply:
This is okay with me.